Dockerfile

```diff
@@ -2,14 +2,18 @@
 #
 # Use of this source code is governed by the LICENSE file in this repository.
 
+# set a global Docker argument for the default CLI version
+#
+# https://github.com/moby/moby/issues/37345
+ARG TERRAFORM_VERSION=0.12.26
 
 ################################################################################
 ##     docker build --no-cache --target binary -t vela-terraform:binary .     ##
 ################################################################################
 
 FROM alpine:latest as binary
 
-ENV TF_VERSION=0.12.24
+ARG TERRAFORM_VERSION
 
 RUN wget -q https://releases.hashicorp.com/terraform/${TF_VERSION}/terraform_${TF_VERSION}_linux_amd64.zip -O terraform.zip && \
```

Comment:
you will want to change `${TF_VERSION}` to `${TERRAFORM_VERSION}` now

Reply:
Didn't even look at that, only looked at the line you commented on 😞

Reply:
my fault. should have called that out as well.

Reply:
I looked at the change without ☕ so tunnel vision sunk in 😬